Build fail with -DGEN8=OFF or -DGEN10=OFF

When building media driver sources from latest release tag "intel-media-20.3.0":
`cmake -G Ninja  -DGEN8=OFF -DGEN9=ON -DGEN10=OFF -DGEN11=ON -DGEN12=ON .  &&  ninja`

The build fails with:
```
In file included from ../media_driver/agnostic/common/cm/cm_command_buffer.cpp:44:
../media_driver/agnostic/gen12/cm/cm_hal_g12.h:31:10: fatal error: mhw_render_hwcmd_g8_X.h: No such file or directory
   31 | #include "mhw_render_hwcmd_g8_X.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

If I change cmake to use "-DGEN8=ON", then the build fails to:
```
../media_driver/agnostic/gen11/codec/hal/codechal_vdenc_vp9_g11.cpp:41:10: fatal error: igcodeckrn_g10.h: No such file or directory
   41 | #include "igcodeckrn_g10.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
```

Aren't you doing automated build-testing before releases to check that the GEN options in CMake still work?

--- a/media_driver/agnostic/gen12/cm/cm_hal_g12.h
+++ b/media_driver/agnostic/gen12/cm/cm_hal_g12.h
@@ -28,7 +28,6 @@
 #define _CM_HAL_G12_H_
 
 #include "cm_hal.h"
-#include "mhw_render_hwcmd_g8_X.h"
 
 #define CM_NUM_HW_POLYPHASE_TABLES_G12         17
 #define CM_NUM_HW_POLYPHASE_EXTRA_TABLES_G12   15

